### DISCUSSION TITLE
iboss exploit

### DISCUSSION THREAD

**Discussion by viperbot2000** [3kh0/ext-remover#599] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://github.com/BlueHatCrew/fsc/blob/main/info.md
Is this possible for gogaurdian and or lightspeed?

**Comment by cooltoo37** [comment]
nope

**Comment by BroJac5246** [comment]
Note that Ingot for iBoss is better for those who do have iBoss. See #585 for more details.

**Comment by Aka-but-nice** [comment]
> https://github.com/BlueHatCrew/fsc/blob/main/info.md Is this possible for gogaurdian and or lightspeed?

Read the document very carefully.
